apps/src/lab2/levelEditors/exemplar/ExemplarSettings.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
+
+import {AppName, ExemplarSettings} from '../../types';
+
+import ExemplarPlayer from './ExemplarPlayer';
+import ExemplarValidation from './ExemplarValidation';
+
+const AppExemplarSupport: {[key in AppName]?: boolean} = {
+  music: true,
+};
```

Comment:
nit: could pull this out as a common value with `ExemplarValidation`, or import one from the other

edit: or, does `ExemplarValidation` need to check this again since it's only rendered if this check passes ni the first place?

Reply:
Good call. It does not need to check it again.